.babelrc.js

```diff
@@ -44,6 +44,7 @@ module.exports = function(api) {
       },
       modules: {
         presets: [['@babel/preset-env', {
+          modules: false,
```

Comment:
It seems counterintuitive, but;
```
@babel/preset-env: The 'modules' option must be one of 
 - 'false' to indicate no module processing
 - a specific module type: 'commonjs', 'amd', 'umd', 'systemjs' - 'auto' (default) which will automatically select 'false' if the current
   process is known to support ES module syntax, or "commonjs" otherwise
```

Meaning by setting it to false, it won't try to process modules into CJS etc.